travis.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+
+build_cpp() {
+  ./autogen.sh
+  ./configure
+  make -j2
+  make distcheck -j2
```

Comment:
Yes, on second thought, I think you're right -- does make sense to split this now. Just pushed an update.